packages/core/src/Engine.ts

```diff
@@ -754,4 +754,10 @@ export interface EngineConfiguration {
   colorSpace?: ColorSpace;
   /** Shader lab */
   shaderLab?: IShaderLab;
+  /** Input options */
+  inputOptions?: {
+    keyboard?: EventTarget;
+    pointer?: EventTarget;
+    wheel?: EventTarget;
```

Comment:
`keyboard`、`pointer`、`wheel` is not clear  

`keyboardTarget` .... maybe is better

Reply:
Done.